src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/SymbolExtensions.cs

```diff
@@ -34,6 +34,25 @@ public static IEnumerable<IEventSymbol> GetAllEvents(this INamedTypeSymbol symbo
 			} while (symbol.SpecialType != SpecialType.System_Object);
 		}
 
+		public static IEnumerable<ISymbol> GetAllMembers(this ITypeSymbol symbol)
+		{
+			do
+			{
+				foreach (var member in symbol.GetMembers())
+				{
+					yield return member;
+				}
+
+				symbol = symbol.BaseType;
+
+				if (symbol == null)
+				{
+					break;
+				}
+
+			} while (symbol.SpecialType != SpecialType.System_Object);
```

Comment:
Seems like this will omit, eg, `ToString()`, is that a problem?

Reply:
Correct. Not in my scenario, but it's not returning all members indeed. Good catch!